src/containers/User/Detail/Credits/helpers.js

```diff
@@ -32,7 +32,7 @@ const getRowDataForPurchase = (purchase) => ({
     <DcrTransactionLink txID={purchase.txId} />
   ),
   Status: purchase.confirming ? (
-    <StatusTag type="bluePending" text="Waiting for confirmations" />
+    <StatusTag type="bluePending" text={`Waiting for confirmations(${purchase.confirmations}/${purchase.minRequiredConfirmations})`} />
```

Comment:
what about adding some space or padding before the first parenthesis?
```suggestion
    <StatusTag type="bluePending" text={`Waiting for confirmations (${purchase.confirmations}/${purchase.minRequiredConfirmations})`} />
```